template/mfcsTemplate/templateHeader.php

```diff
@@ -17,13 +17,13 @@
 
 
 		<!-- Favicon Information -->
-				<link rel="apple-touch-icon" sizes="180x180" href="images/apple-touch-icon.png">
-				<link rel="icon" type="image/png" href="images/favicon-32x32.png" sizes="32x32">
-				<link rel="icon" type="image/png" href="images/favicon-16x16.png" sizes="16x16">
-				<link rel="manifest" href="images/manifest.json">
-				<link rel="mask-icon" href="images/safari-pinned-tab.svg" color="#5bbad5">
-				<link rel="shortcut icon" href="images/favicon.ico">
-				<meta name="msapplication-config" content="images/browserconfig.xml">
+				<link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon.png">
+				<link rel="icon" type="image/png" href="/images/favicon-32x32.png" sizes="32x32">
+				<link rel="icon" type="image/png" href="/images/favicon-16x16.png" sizes="16x16">
+				<link rel="manifest" href="/images/manifest.json">
+				<link rel="mask-icon" href="/images/safari-pinned-tab.svg" color="#5bbad5">
+				<link rel="shortcut icon" href="/images/favicon.ico">
+				<meta name="msapplication-config" content="/images/browserconfig.xml">
```

Comment:
This should be in a separate pull request. Putting it in the same one means that if there is an issue with the public/oai check box pull request and it needs reverted to fix a problem, then this change will also be reverted out. Please separate into a separate pull request.